app/internal/window/os_windows.go

```diff
@@ -578,6 +588,48 @@ func (w *window) writeClipboard(s string) error {
 	return nil
 }
 
+func (w *window) SetOption(option Option) {
+	if option.Title != nil {
+		windows.SetWindowText(w.hwnd, *option.Title)
+		w.opts.Title = option.Title
+	}
+	if option.Height != nil && option.Width != nil {
```

Comment:
Check for Height and Width != nil individually. Or perhaps better, combine Width and Height to a single field, Size.